### PR TITLE
Use autocomplete_select for granter and grantee

### PR DESCRIPTION
**Pull request by mllocs**
Fixes https://github.com/coopdevs/decidim-module-action_delegator/issues/7

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -24,6 +24,9 @@ en:
               title: General information
             save: Create
             title: New delegation
+            select_member: Select member
+            grantee: Grantee
+            granter: Granter
```
Comment (mllocs) [inline config/locales/en.yml:29]: I could use the ones in index instead

**Comment by microstudi** [comment]
Hi @mllocs , welcome here!

You need to fix lint errors, you can run these commands to fix them:

```ruby
bundle exec rubocop -a
bundle exec erblint app/**/*.erb --autocorrect        
```

**Comment by mllocs** [comment]
Hello @microstudi!

Thanks for the tip. Now there's just a test failing, I need to change how we populate the granter and grantee autocompleters in tests. There's any helpers to do that in the capybara tests?

**Comment by sauloperez** [comment]
looks like https://github.com/decidim/decidim/blob/7a4b0719d4016d9af0b955e7e5eea8b0ab92394a/decidim-assemblies/spec/shared/manage_assembly_members_examples.rb#L51 is what we are looking for.

File: config/locales/en.yml (hunk 1 above)
Comment (sauloperez) [reply config/locales/en.yml:26]: I think the duplication in this case is fine. It gives us flexibility

**Review by sauloperez — APPROVED** [review]

**Pull request merged**